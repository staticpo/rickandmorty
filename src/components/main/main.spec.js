import React from 'react';
import { shallow, render, mount } from 'enzyme';
import escapeSnapshot from '../../../tests/commons';
import Main from './index';

describe('Main (Snapshot)', () => {
  const defaultProps = {};

  it('renders with default props', () => {
    const component = shallow(<Main {...defaultProps} />);
    expect(escapeSnapshot(component)).toMatchSnapshot();
  });
});
