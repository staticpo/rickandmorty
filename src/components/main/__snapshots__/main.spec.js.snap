// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main (Snapshot) renders with default props 1`] = `
"<div>
  <section className='info' />
  <Paginator eventHandler={[Function: bound switchPage]} currentPage={1} pages={0} next={[undefined]} prev={[undefined]} />
</div>"
`;
